src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveReviewsServiceTest.kt

```diff
@@ -198,11 +210,110 @@ class IncentiveReviewsServiceTest {
           positiveBehaviours = 0,
           negativeBehaviours = 0,
           acctOpenStatus = false,
+          nextReviewDate = iepTime.toLocalDate().plusYears(1),
         ),
       )
     )
   }
 
+  @Test
+  fun `oldest date of next review is first`(): Unit = runBlocking {
+    // Given
+    val prisonerNumber = "G6123VU"
+    whenever(prisonApiService.getLocation(any())).thenReturnLocation("MDI-2-1")
+    whenever(offenderSearchService.findOffenders(any(), any(), any(), any()))
+      .thenReturnOffenders(
+        listOf(
+          offenderSearchPrisoner(prisonerNumber, "110001"),
+          offenderSearchPrisoner(prisonerNumber, "110002")
```

Comment:
minor thing, but it'd make sense to have different prisoner numbers as well as different booking ids. i know that won't affect the calculations of what these tests are actually asserting, but the tests would read more like reality

Reply:
Good point - addressed https://github.com/ministryofjustice/hmpps-incentives-api/pull/237/commits/0f7365b19d018a307509345d6d2bcbfacfa0485e